src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -17,6 +17,7 @@ class Job {
     DateTime startedTimestamp
     DateTime finishedTimestamp
     Status status
+    String statusLink
```

Comment:
I'm not really sure that it's good to store this in here, because the link is not *really* a property of the job (it's more to do with how the app is configured and what not).  Could you not generate the link on the fly in `JobPresenter`, e.g. add a `addUrl` method to `JobPresenter` (the same way as `addAggrUrl` does it)?  This is better IMO because it keeps `Job` with only its real properties and leaves URLs and other stuff needed for the client to a class dedicated to that...

In fact, I thought this is what you were going to do after talking on Fri, so maybe there's a reason you couldn't do it this way?